clients/http/command.go

```diff
@@ -67,6 +67,20 @@ func (client *CommandClient) IssueGetCommandByName(ctx context.Context, deviceNa
 	return res, nil
 }
 
+func (client *CommandClient) IssueGetCommandByNameWithQueryParams(ctx context.Context, deviceName string, commandName string, queryParams map[string]string) (res *responses.EventResponse, err errors.EdgeX) {
+	requestParams := url.Values{}
+	for k, v := range queryParams {
+		requestParams.Set(k, v)
+	}
+
+	requestPath := path.Join(common.ApiDeviceRoute, common.Name, url.QueryEscape(deviceName), url.QueryEscape(commandName))
+	err = utils.GetRequest(ctx, &res, client.baseUrl, requestPath, requestParams)
```

Comment:
Is it ok that the `dsPushEvent`  and `dsReturnEvent` parameters are not sent. I.e. does the Device Service have a default for these if not present? If not, we need to add defaults here is they are not present in the passed in `queryParams`

Reply:
@lenny-intel since we have the default value in core-command, I don't think we should add the default value in the client library to prevent the double maintenance effort if we want to change the default values in the future.

Reply:
Device service also have default value:
https://docs.edgexfoundry.org/2.0/design/adr/device-service/0011-DeviceService-Rest-API/#query-parameters